shared/uavobjectdefinition/mixersettings.xml

```diff
@@ -30,19 +30,19 @@
 				<option>CameraRoll</option>
 				<option>CameraPitch</option>
 				<option>CameraYaw</option>
-				<option>Accessory0</option>
-				<option>Accessory1</option>
-				<option>Accessory2</option>
 			</options>
 		</field>
-		<field name="Mixer1Vector" units="" type="int8" defaultvalue="0">
+		<field name="Mixer1Vector" units="" type="int16" defaultvalue="0">
 			<description>Mixer vector</description>
```

Comment:
A brief description of what the values mean would be super nice (e.g. 128 = blah blah)